ec2/sc-ec2-windows-jumpcloud-development.yaml

```diff
@@ -0,0 +1,354 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This is the development product -- scroll down for the real version.

Reply:
would be better to add this dev product as a separate PR.  maybe a follow on PR after you add ec2/sc-ec2-windows-jumpcloud-v1.0.0.yaml ?

Reply:
Now that the PR has been reviewed, why would it be better? For the sake of the history?

Reply:
yes, history, reverting, generally easier to understanding and review.   i actually haven't reviewed this file at all.